bgpd/bgp_routemap.c

```diff
@@ -3946,6 +3972,32 @@ DEFUN (no_match_origin,
 				      RMAP_EVENT_MATCH_DELETED);
 }
 
+DEFUN (set_table_id,
+	set_table_id_cmd,
+	"set table (1-4294967295)",
+	SET_STR
+	"export route to non-main kernel table\n"
+	"Kernel routing table id\n")
+{
+	int idx_id = 2;
+
+	VTY_DECLVAR_CONTEXT(route_map_index, index);
+
+	return generic_set_add(vty, index, "table", argv[idx_id]->arg);
+}
+
+DEFUN (no_set_table_id,
+	no_set_table_id_cmd,
+	"no set table",
```

Comment:
We need to allow the cli line where the user specifies the table that they don't want.

Reply:
I don't understand the need you mention.
From my point of view, the user handles set table <> to redirect entries to a specific table, instead of the default one.
What is in this pull request does not cover the case where the initial bgp entries are in a specific table, and route-map explicitly does not want the bgp entries to be exported in that bgp table. also, what would be the consequence of that ? the entries would be in default table ?

Reply:
I think @donaldsharp had `no set table [(1-4294967295)]` in his mind :)

Reply:
Hi @ton31337 , actually every route-map entry can set a specific table id.
and there is a separate vty command that cancels that behaviour ( no set table_id[]).

I don't plan for now to export entries in multiple table entries :-).

Reply:
makes sense :)